apps/site/pages/docs/headless-cms-integration/2-setting-up-vtex-headless-cms.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: '2. Setting up the VTEX Headless CMS in your FastStore project'
+sidebar_label: 'headless-cms-integration'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+# Part 2: Setting up the VTEX Headless CMS in your FastStore project
+</header>
+
+Now that you are familiar with the core concepts of the [**VTEX Headless CMS**](/docs/headless-cms-overview/overview-headless-cms) and have [set it up in your VTEX account](/docs/headless-cms-integration/1-configuring-the-vtex-account), let's get back to our **FastStore** project to define our own **Content Types** and **Sections**.
+
+By the end of this part of this tutorial, you will be able to see the definition of a Content Type available for use at the VTEX Headless CMS app. 
+
+---
+
+## Step by step
+
+### Step 1 - Creating the CMS folder
+
+1. Open the terminal and change to the root directory of your FastStore project.
+2. Create a new folder named `cms` at the root of your FastStore project:
+
+    ```bash
+    mkdir cms
+    ```
+
+3. Inside the `cms` folder, create a folder for your project:
+
+    ```bash
+    mkdir cms/faststore
```

Comment:
This command doesn't work for Windows users, the correct one for them is: mkdir cms\faststore